src/test/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributorTest.java

```diff
@@ -85,10 +80,18 @@ public void setUp() throws Exception {
     actionCase = new ActionCase();
     actionCase.setSampleUnitType("H");
 
+    MockitoAnnotations.initMocks(this);
+    DataGrid dataGrid = new DataGrid();
+    dataGrid.setLockTimeToLiveSeconds(30);
+    dataGrid.setLockTimeToWaitSeconds(600);
+    when(appConfig.getDataGrid()).thenReturn(dataGrid);
+    ActionDistribution actionDistribution = new ActionDistribution();
+    actionDistribution.setRetrievalMax(1000);
+    when(appConfig.getActionDistribution()).thenReturn(actionDistribution);
+
     lock = mock(RLock.class);
     when(redissonClient.getFairLock(any())).thenReturn(lock);
-
-    MockitoAnnotations.initMocks(this);
+    when(lock.tryLock(anyLong(), anyLong(), any(TimeUnit.class))).thenReturn(true);
```

Comment:
you probably want to `.verify` that `.tryLock` and `.unlock` are actually called